<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>QuantumPointContact.png</normaloff>QuantumPointContact.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sweepParameters_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>639</y>
      <width>601</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>112</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="checkbox_verticalLayout">
      <item>
       <widget class="QCheckBox" name="gs610R_checkBox">
        <property name="text">
         <string>GS610 (R)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gs610L_checkBox">
        <property name="text">
         <string>GS610 (L)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gs210_checkBox">
        <property name="text">
         <string>GS210</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="magnet_checkBox">
        <property name="text">
         <string>Magnet (T)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="time_checkBox">
        <property name="text">
         <string>Time (s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>471</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="gs610RInput_horizontalLayout">
        <item>
         <widget class="QLineEdit" name="gs610RStart_lineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="gs610RLoop_comboBox">
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5-&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610RStop_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610RCurrent_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610RPoints_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610RStep_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="magnetInput_horizontalLayout">
        <item>
         <widget class="QLineEdit" name="magnetStart_lineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="magnetLoop_comboBox">
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5-&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="magnetStop_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="magnetCurrent_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="magnetPoints_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="magnetStep_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="gs210Input_horizontalLayout">
        <item>
         <widget class="QLineEdit" name="gs210Start_lineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="gs210Loop_comboBox">
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5-&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gs210Stop_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs210Current_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs210Points_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs210Step_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="gs610LInput_horizontalLayout">
        <item>
         <widget class="QLineEdit" name="gs610LStart_lineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="gs610LLoop_comboBox">
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5-&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610LStop_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610LCurrent_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610LPoints_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="gs610LStep_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="timeInput_horizontalLayout">
        <item>
         <widget class="QLineEdit" name="timeStart_lineEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="timeLoop_comboBox">
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4-&gt;</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5-&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeStop_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="timeCurrent_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="timePoints_lineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="timeStep_lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="backPoints_lineEdit">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>233</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="backPoints_label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>237</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back Points:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sweepBack_checkBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>237</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sweep Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepStart_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepCurrent_label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepPoints_label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepLoop_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepStop_label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepStep_label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Step</string>
     </property>
    </widget>
    <widget class="QLabel" name="sweepLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sweep Parameters</string>
     </property>
    </widget>
    <zorder>gridLayoutWidget</zorder>
    <zorder>backPoints_lineEdit</zorder>
    <zorder>backPoints_label</zorder>
    <zorder>sweepBack_checkBox</zorder>
    <zorder>sweepStart_label</zorder>
    <zorder>sweepCurrent_label</zorder>
    <zorder>sweepPoints_label</zorder>
    <zorder>sweepLoop_label</zorder>
    <zorder>sweepStop_label</zorder>
    <zorder>sweepStep_label</zorder>
    <zorder>sweepLabel</zorder>
    <zorder>verticalLayoutWidget</zorder>
   </widget>
   <widget class="QFrame" name="fileSave_frame">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>639</y>
      <width>370</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="folderPath_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>350</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="fileSave_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectFolder_pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Folder</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="save_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>97</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="comment_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>80</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="comment_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>350</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="saveControlState_checkBox">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>36</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Control State</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="run_frame">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>639</y>
      <width>360</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="errorOut_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>340</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="errorOut_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>80</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Out:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberOfCurves_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>150</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearMemory_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Memory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="run_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="numberOfCurves_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>190</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Number Of Visible Curves:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="figures_tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1350</width>
      <height>620</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="li5650">
     <attribute name="title">
      <string>LI5650 X And Y</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>LI5650 R And Theta</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_5"/>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>SR830 X And Y</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_6"/>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>SR830 R And Theta</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_7"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuInstrument">
    <property name="title">
     <string>Instrument</string>
    </property>
    <addaction name="control_action"/>
    <addaction name="measure_action"/>
   </widget>
   <addaction name="menuInstrument"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSet_Up">
   <property name="text">
    <string>Set Up</string>
   </property>
  </action>
  <action name="actionConnect_2">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect_2">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSet_Up_2">
   <property name="text">
    <string>Set Up</string>
   </property>
  </action>
  <action name="actionConnect_3">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect_3">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="control_action">
   <property name="text">
    <string>Control</string>
   </property>
  </action>
  <action name="measure_action">
   <property name="text">
    <string>Measure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
